org.darkdimension.rue.json

```diff
@@ -0,0 +1,53 @@
+{
+	"id": "org.darkdimension.rue",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "rue",
+	"modules": [
+		"shared-modules/glew/glew.json",
+		"shared-modules/glu/glu-9.json",
+		{
+			"name": "avdl",
+			"buildsystem": "simple",
+			"build-commands": [
+				"make prefix=",
+				"make DESTDIR=${FLATPAK_DEST} prefix= install"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/tomtsagk/avdl/archive/refs/tags/v0.1.3.tar.gz",
+					"md5": "3d4fea5a6f4b1e1856f2aa4b82b55250"
```

Comment:
please use sha256 instead

Reply:
I've updated the `md5` field to use the `sha256` sum instead.